### ISSUE TITLE
Creative mode feature request

### ISSUE THREAD

**Issue by Claysive** [mircearoata/SatisfactoryPakUtilityMod#5]
In old versions, creative mode and no cost mode used to make it where items in the craft bench and work station were no cost as well. This version seems to have dropped that. Would be another QOL function that would be nice if it were added back(if possible). Thanks!

**Comment by mircearoata** [comment]
This is a completely different mod, than UtilityMod, because the old one wasn't updated anymore I decided to do this mod which shouldn't need often updating like the other did.

Neither of the mods ever allowed no cost in the workbench, because both of them just enable the built-in no cost cheat, that Coffee Stain programmed. So if it did do workbench no cost it was because Coffee Stain made it do that and they removed it, but I doubt that, because I remember people asking why does it cost to make items in the workbench at the time old UtilityMod was still working

**Comment by TrustyKdawg** [comment]
I think the confusion with no cost in the workbench is that if you had the items in your inventory, it took them to make with but if you didn't it still made and gave you the item.

**Comment by Claysive** [comment]
Yeah, I get that maybe it has something to do with Coffee Stain, but the old one by brabb3l 100% used to allow no cost in the workbench and crafting table. It was the only mod I used back then and on new play throughs, the first thing I would do is drop down a bench and make all of my essentials even with nothing in my inventory. 

I wasn't aware that Coffee Stain has their own built in no cost cheat and that the mod just enables that so I do see where you are coming from that it could just be an issue on their part. Just a feature that I miss because no cost should be no cost all the way across the board. Anyways, thanks for the response and I hope maybe this will get fixed one day.

**Comment by mircearoata** [comment]
I asked the old UtilityMod's dev and he said that he indeed added the extra workbench nocost after toggling the built in nocost. Unfortunately, it was done by hooking a function to always allow items to be crafted, and this can not be done in a pak only mod which is the purpose of this mod
